deploy/helm/flightctl/Chart.yaml

```diff
@@ -10,6 +10,6 @@ dependencies:
     version: "0.0.1"
   - name: flightctl-ui
     condition: ui.enabled
-    version: "0.3.0-rc1"
+    version: "0.3.0-rc2"
```

Comment:
Should be 0.3.1.rc2

Reply:
why? we have not released 0.3.0 yet :)

This is still "I am trying to release 0.3.0", my 2nd candidate. At least for all the Red Hat standards I have been exposed too. I understand this could be different in other companies.

Reply:
Yup. It's the second _candidate_ that might be tagged 0.3.0 eventually, i.e. 0.2.0 < 0.3.0-rc1 < 0.3.0-rc2 < or = 0.3.0.

Reply:
We want to release 0.3.0, and we will be tagging as many release candidates for QE to test as needed, if you change the version, the scope should be different, and we're not introducing features, only bugfixes. So I think it's just 0.3.0-rc2.